samza-sql/src/main/java/org/apache/samza/sql/runner/SamzaSqlApplicationConfig.java

```diff
@@ -82,6 +85,8 @@
 
   public static final String CFG_GROUPBY_WINDOW_DURATION_MS = "samza.sql.groupby.window.ms";
 
+  private static final String LOG_OUTPUT_STREAM = "log.outputStream";
```

Comment:
This is dup from SamzaSqlDslConverter's static var definition. Can we move it to a common place?

Reply:
yes.